lib/views/landing_screen.dart

```diff
@@ -1,13 +1,24 @@
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:go_router/go_router.dart';
 
 // widgets
 import 'auth/widgets/login_signup_widgets.dart';
 
-class LandingPage extends StatelessWidget {
+class LandingPage extends ConsumerStatefulWidget {
```

Comment:
Where is this widget using a provider? If it was using one before, why wasn't it a ConsumerStatefulWidget before?

Reply:
hmm i probably changed it to consumer state to fix a bug but it didn't work so I forgot to change it back. I'll change it in the next pull request